src/app/core/config/config.service.ts

```diff
@@ -35,7 +36,7 @@ export class ConfigService extends LatestEntityLoader<Config> {
   }
 
   public getConfig<T>(id: string): T {
-    return this.currentConfig.data[id];
+    return this.applyMigrations(id, this.currentConfig.data[id]);
```

Comment:
Same as in the other PR. Only patching this ad-hoc does not allow to save the full config later.

Reply:
to be honest, I am not sure why I opted for migrating during getConfig instead of loading of config from DB. I'll change this as you suggest